definitions/reports/alternate_content_sources.rb

```diff
@@ -0,0 +1,75 @@
+module Checks
+  module Report
+    class AlternateContentSources < ForemanMaintain::Report
+      metadata do
+        description 'Facts about ACSs'
+        confine do
+          feature(:katello)
+        end
+      end
+
+      def run
+        data_field('custom_alternate_content_sources_count') { custom_alternate_content_sources }
+        data_field('simplified_alternate_content_sources_count') do
+          simplified_alternate_content_sources
+        end
+        data_field('rhui_alternate_content_sources_count') { rhui_alternate_content_sources }
+        data_field('yum_alternate_content_sources_count') { yum_alternate_content_sources }
+        data_field('file_alternate_content_sources_count') { file_alternate_content_sources }
+      end
+
+      def custom_alternate_content_sources
+        query =
+          query(
+            <<-SQL
+              SELECT count(*) as custom_acs_count FROM "katello_alternate_content_sources"
+                WHERE "katello_alternate_content_sources"."alternate_content_source_type" = 'custom'
+            SQL
+          )
+        query.first['custom_acs_count'].to_i
```

Comment:
You could simplify this by using the `sql_count` helper[1]. Something along the lines of this should work

```suggestion
        sql_count("katello_alternate_content_sources WHERE alternate_content_source_type = 'custom'")
```

[1] - https://github.com/theforeman/foreman_maintain/blob/02ce1e30bda950701b70e1aa4d65812ad5cbec46/lib/foreman_maintain/report.rb#L14